src/current/_includes/releases/v23.1/v23.1.12.md

```diff
@@ -0,0 +1,168 @@
+- Requests for database details or table details from the UI, or usages of `SHOW RANGES WITH DETAILS` are no longer subject to errors if the number of requested spans is too large. [#109902][#109902]
+- Added a new [metric](https://www.cockroachlabs.com/docs/v22.2/metrics) `changefeed.lagging_ranges` that shows the number of [ranges](https://www.cockroachlabs.com/docs/v22.2/architecture/overview#architecture-range) which are behind in changefeeds. This metric can be used with the [`metrics_label` changefeed option](https://www.cockroachlabs.com/docs/v22.2/create-changefeed#options). The calculation of this metric is controlled by 2 new [cluster settings](https://www.cockroachlabs.com/docs/v22.2/cluster-settings): (1) `changefeed.lagging_ranges_threshold`, with a default of 3 minutes, is the amount of time a range needs to be behind to be considered lagging, and (2) `changefeed.lagging_ranges_polling_interval`, with a default of 1 minute, controls how often the lagging ranges calculation is done. Note that polling adds latency to the metric being updated. For example, if a range falls behind by 3 minutes, the metric may not update until an additional minute afterwards. Also note that ranges undergoing an initial scan for longer than the threshold are considered to be lagging. Starting a changefeed with an initial scan on a large table will likely increment the metric for each range in the table. However, as ranges complete the initial scan, the number of lagging ranges will decrease. [#110963][#110963]
+- The [DB Console](https://www.cockroachlabs.com/docs/v23.1/ui-overview) now constructs client-side requests using relative URLs instead of absolute ones. This enables proxying of the DB Console at arbitrary subpaths. [#111652][#111652]
+- Added a new [cluster setting](https://www.cockroachlabs.com/docs/v23.1/cluster-settings) `server.http.base_path` that controls the redirection of the browser after successful login with [OIDC SSO](https://www.cockroachlabs.com/docs/v23.1/sso-db-console). It is unlikely that this setting would need adjustment. However, it is helpful in cases where CockroachDB is running behind a load balancer or proxy that serves CockroachDB under a subpath, such as `https:// <hostname>/crdb/ `. In those cases, it is necessary for the browser to redirect to `/ crdb` after login instead of `/`, which has always been the hard-coded default.[#112038][#112038]
```

Comment:
```suggestion
- Added a new [cluster setting](https://www.cockroachlabs.com/docs/v23.1/cluster-settings) `server.http.base_path` that controls the redirection of the browser after successful login with [OIDC SSO](https://www.cockroachlabs.com/docs/v23.1/sso-db-console). Most users do not need to modify this setting. However, it is helpful in cases where CockroachDB is running behind a load balancer or proxy that serves CockroachDB under a subpath, such as `https:// <hostname>/crdb/ `. In those cases, it is necessary for the browser to redirect to `/ crdb` after login instead of `/`, which has always been the hard-coded default.[#112038][#112038]
```